manticore/core/manticore.py

```diff
@@ -356,7 +370,60 @@ def __init__(self, initial_state, workspace_url=None, outputspace_url=None, **kw
 
         # Workers will use manticore __dict__ So lets spawn them last
         self._workers = [self._worker_type(id=i, manticore=self) for i in range(consts.procs)]
-        self._is_main = True
+        self._snapshot = None
+        self._main_id = os.getpid(), threading.current_thread().ident
+
+    def is_main(self):
```

Comment:
Not a big deal, but this could be a `@property` so as to preserve the old API (not that anyone uses it...)

Reply:
I'M going towards the rule: if the thing is an  "is_something" then it is a method and not a property.
I fear someone use something like
```
 if x.is_main:
     do_stuff()
```

When it is actually a method. Thoughts?

Reply:
I couldn't find great documentation on when it's "pythonic" to use properties over methods, other than [one SO commenter](https://stackoverflow.com/questions/1554546/when-and-how-to-use-the-builtin-function-property-in-python) who argues that you should use properties when you're committed to not having any side-effects in a method. Personally, `is_main` feels to me like it's tautologically describing a "property" of the Manticore object, and so it makes sense to use properties here. That said, I don't feel too strongly about it, so don't feel compelled to change it if you disagree.